src/develop/masks/path.c

```diff
@@ -1339,7 +1339,7 @@ static int _path_events_button_pressed(struct dt_iop_module_t *module,
       dt_dev_masks_selection_change(darktable.develop, crea_module, form->formid);
       gui->creation_module = NULL;
 
-      if(gui->creation_continuous)
+      if(crea_module && gui->creation_continuous)
```

Comment:
@TurboGit I thought it made more sense to have the check for `crea_module` in the next if statement

Reply:
But this will enable continuous creation of path in mask manager which is not supported for circles & ellipses.

Reply:
Shouldn't that work though? (i.e. shouldn't the fix be with circles & ellipses?)

Reply:
Ok, it works for circle, ellipse, path & gradient now. I'll do the same for brush when your fixes will be in place.